src/child/spawn-child.ts

```diff
@@ -10,14 +10,17 @@ import { argPrefix, compress } from './argv-payload';
  *   the child process.
  */
 export function callInChild(state: BootstrapState) {
+  const loaderURL = pathToFileURL(require.resolve('../../child-loader.mjs'));
+  loaderURL.searchParams.set(argPrefix, compress(state));
+
   const child = spawn(
     process.execPath,
     [
       '--require',
       require.resolve('./child-require.js'),
       '--loader',
       // Node on Windows doesn't like `c:\` absolute paths here; must be `file:///c:/`
-      pathToFileURL(require.resolve('../../child-loader.mjs')).toString(),
+      loaderURL.toString(),
       require.resolve('./child-entrypoint.js'),
       `${argPrefix}${compress(state)}`,
```

Comment:
Note to self: can we include state only once to reduce the risk we hit `CommandLine` length limit on Windows?
Consider how this will look with forthcoming `register()` API.  If `register()` will allow sending the state payload in-memory, akin to current `lateBind` logic, then consider using a pattern that looks similar today.

Reply:
Yeah good call, I guess child-entrypoint should just pull it from the loader URL if it needs it.

Reply:
Ah, so the tricky bit here is that the child-entrypoint is modifying and re-stashing the value in the state's argv for child processes, so storing it _only_ in the loader url execArgv involves a bit more refactoring. Can definitely be done, but I'd recommend putting it off for a second commit (if not second PR) just to ensure edge cases are handled properly.